Automattic/sensei
# Improve styles for sensei login page

Resolves https://github.com/Automattic/sensei/issues/6368

## Proposed Changes
* Improved the style of the login page following the Woocommerce login
* Improved the alert message shown when there's a login error
* Fixed the issue of "My Messages" button getting rendered even for logged-out users

## Testing Instructions


1. Log out
2. Go to My Courses page from frontend
3. It should look better than before and nothing should be broken
4. Try with all variations of the Course theme, Divi and Astra
5. Enable registration from Settings -> General -> Anyone can Register
6. Make sure it looks better with the Registration form too
7. Check in mobile view as well for all the above
8. Check the error alert with a wrong password

#### Course theme:
<img width="1113" alt="Screenshot 2023-10-05 at 4 32 17 PM" src="https://github.com/Automattic/sensei/assets/6820724/6f39930b-8265-418b-82c1-873d7e4f336e">
<img width="1113" alt="Screenshot 2023-10-05 at 4 32 37 PM" src="https://github.com/Automattic/sensei/assets/6820724/88c6eaf3-79ab-4fde-a779-49ace8df16e2">
<img width="1113" alt="Screenshot 2023-10-05 at 4 32 52 PM" src="https://github.com/Automattic/sensei/assets/6820724/4e722634-f628-422d-98f9-b23b5073ed68">
<img width="1113" alt="Screenshot 2023-10-05 at 4 33 09 PM" src="https://github.com/Automattic/sensei/assets/6820724/ac70dfcc-90fa-4601-9c49-ba0b64c7b823">

#### Divi
<img width="1191" alt="Screenshot 2023-10-05 at 4 34 43 PM" src="https://github.com/Automattic/sensei/assets/6820724/dc9bc27f-993e-49ed-8af3-913f1f749c02">

#### Astra
<img width="1249" alt="Screenshot 2023-10-05 at 4 34 07 PM" src="https://github.com/Automattic/sensei/assets/6820724/a5a4fb6a-ef42-429d-a508-84c10958af21">

#### Mobile
<img width="523" alt="Screenshot 2023-10-05 at 4 35 34 PM" src="https://github.com/Automattic/sensei/assets/6820724/cc1b83ad-b6a3-4be1-98a0-160b3b47ed0d">

#### Error message
![image (2)](https://github.com/Automattic/sensei/assets/6820724/0029010f-b3b0-4fea-a504-ce4774935912)

## Pre-Merge Checklist
